pinecone/data/index.py

```diff
@@ -103,6 +111,23 @@ def __init__(
             api_version=API_VERSION,
         )
 
+        self._load_plugins()
+
+    def _load_plugins(self):
+        """@private"""
+        try:
+            # I don't expect this to ever throw, but wrapping this in a
+            # try block just in case to make sure a bad plugin doesn't
+            # halt client initialization.
+            openapi_client_builder = build_plugin_setup_client(
+                config=self.config,
```

Comment:
I think this needs to be `self._config`. Currently seeing this error when it attempts to apply plugins to `Index`:

```
ERROR    pinecone.data.index:index.py:129 Error loading plugins in Index: 'Index' object has no attribute 'config'
```